backend/fixtures/useFixtures.js

```diff
@@ -40,7 +38,5 @@ export const useFixtures = () => {
     SyncState.update(name)
   })
 
-  runRemap(remap)
```

Comment:
This is removed, because it caused runRemap to run BEFORE syncs with content server had been complete, which caused some errors when syncing docs